index.html

```diff
@@ -243,26 +243,68 @@ <h3>Sampling and Reporting Rate</h3>
+    The telemetry information provided by a [=pressure source=] is represented
+    in this specification as a <dfn>pressure source sample</dfn>, a [=struct=]
+    consisting of the following [=struct/items=]:
+    <ul>
+      <li>
+        <dfn data-dfn-for="pressure source sample">data</dfn>: [=contributing
+        factors=] obtained from the underlying hardware or operating system.
+      </li>
+      <li>
+        <dfn data-dfn-for="pressure source sample">timestamp</dfn>: the
+        [=unsafe shared current time=] when [=pressure source sample/data=] was
+        obtained.
+        <aside class="note">
+          <p>
+            The purpose of [=pressure source sample/timestamp=] is to use the
+            same [=monotonic clock/unsafe current time=] for a sample across
+            all globals and invocations of the [=data collection=] algorithm
+            which are processing the same [=pressure source sample=].
+          </p>
+        </aside>
+      </li>
+    </ul>
   </p>
   <p>
-    A [=platform collector=] can support telemetry for different <dfn>source types</dfn> of computing
-    devices defined by {{PressureSource}}, or there can be multiple [=platform collectors=].
+    A <dfn>platform collector</dfn> is an abstract interface responsible for
+    obtaining telemetry samples from a [=pressure source=], translating them
+    into [=pressure states=] and providing them to the [=user agent=].
   </p>
   <p>
-    From the implementation perspective [=platform collector=] can be treated as a software proxy for the
-    corresponding hardware counters. It is possible to have multiple [=platform collector=] simultaneously
-    interacting with the same underlying hardware if the underlying platform supports it.
+    A [=platform collector=] has the following associated data:
+  </p>
+  <ul>
+    <li>
+      an <dfn data-dfn-for="platform collector">associated pressure
+      source</dfn>, which is a [=pressure source=] or null.
+    </li>
+    <li>
+      an <dfn data-dfn-for="platform collector">activated</dfn> boolean flag,
+      initially false.
+    </li>
+  </ul>
+  <p>
+    The format of the data provided by a [=pressure source=] is
+    [=implementation-defined=], and so is the process through which a
+    [=platform collector=] translates it into a [=pressure state=].
   </p>
   <p>
-    In simple cases, a [=platform collector=] represents individual hardware counters, but if the provided
-    counter readings are a product of data fusion performed in software, the [=platform collector=]
-    represents the results of the data fusion process. This may happen in user space or in kernel space.
+    For this specification's purposes, [=platform collectors=] are scoped to a
+    [=global object=] via the [=platform collector mapping=].
+  </p>
+  <p>
+    For automation purposes, a [=platform collector=] must have the ability to
+    connect to [=virtual pressure sources=] and use their simulated data
```

Comment:
remove data? "data pressure states" sounds weird

Reply:
There was an "as" missing here. Fixed in bc3f3cd3a3e563f666ca01d7ad47e70ab50787ab